test-map.cpp

```diff
@@ -207,21 +207,17 @@ class Test {
     MapStrObj * mss = new MapStrObj();
     Object** keys = mss->get_keys();
     Object** values = mss->get_values();
-    t_true(keys != nullptr);
-    t_true(values != nullptr);
+    t_true(keys == nullptr);
+    t_true(values == nullptr);
 
-    mss->put(s, t);
```

Comment:
A previous team wanted these deletes here. Not sure the reason, but I'm ok with doing this.